parent/pom.xml

```diff
@@ -243,11 +243,13 @@
 					<!-- set useUIThread=false for swtbot tests -->
 
 					<includes>
-						<include>**/AllTests.class</include>
-						<include>**/*AllTests*.class</include>
-						<include>**/*AllBotTests*.class</include>
-						<include>**/*TestSuite*.class</include>
+						<include>**/*Test*.class</include>
+						<include>**/*Test.class</include>
 					</includes>
+					<excludes>
+						<exclude>**/*Abstract*.class</exclude>
+						<exclude>**/$*.class</exclude>
```

Comment:
OK, so TL;DR: 

Assuming you want anything at all written (rather than just relying on upstream defaults), then you want:

Include = **/Test*.java **/*Test.java **/*TestCase.java

Exclude = **/*$*